ProgrammingTask/client.py

```diff
@@ -124,6 +124,9 @@ def main():
 
     microseconds = 0
 
+    p2Flag = False

```

Comment:
Small but important note, `p2Flag` is quite ambiguous name for an attribute. Maybe `player2_connected` or something like that?

Reply:
Please merge this after resolving how you see best!

Reply:
Good call, changed to be more descriptive